test/hooks.js

```diff
@@ -1,10 +1,73 @@
 const { expect } = require( "chai" );
```

Comment:
Could you add `/* eslint-disable no-console */` to the top of this file to suppress the eslint warnings about the use of `console.log` in this file?